tests/upgrades/test_satellitesync.py

```diff
@@ -0,0 +1,136 @@
+
+        :steps:
+
+        1. Before upgrade, Create a ContentView
+        2. Publish and promote the Content View
+
+        :expectedresults: content view should be published and promoted
+        """
+        exporting_cv_name = gen_string('alphanumeric')
+        exporting_prod_name = gen_string('alphanumeric')
+        exporting_repo_name = gen_string('alphanumeric')
+        org = entities.Organization().create()
+        product = entities.Product(organization=org, name=exporting_prod_name).create()
+        repo = entities.Repository(
+            product=product, name=exporting_repo_name,
+            mirror_on_sync=False, download_policy='immediate').create()
+        repo.sync()
+        cv = entities.ContentView(name=exporting_cv_name, organization=org).create()
+        cv.repository = [repo]
+        cv.update(['repository'])
+        cv.publish()
+        cv = cv.read()
+        self.assertTrue(cv.version[0].read().package_count > 0)
+        scenario_facts = {self.__class__.__name__: {
+            'exporting_orgid': org.id,
+            'exporting_cvname': exporting_cv_name,
+            'exporting_prodname': exporting_prod_name,
+            'exporting_reponame': exporting_repo_name}}
+        create_dict(scenario_facts)
+
+    @post_upgrade
+    def test_post_version_cv_export_import(self):
+        """Export and Import cv version created before upgrade
+
+        :steps: Import and Export the Content Views version created before upgrade
+
+        :expectedresults: Content-view created before upgrade should be exported and imported
+            after upgrade
+         """
+        prescene_dict = get_entity_data(self.__class__.__name__)
+        exporting_cv = entities.ContentView(organization=prescene_dict['exporting_orgid']).search(
+            query={'search': 'name={}'.format(prescene_dict['exporting_cvname'])})[0]
+        exporting_cvv_id = max([cvv.id for cvv in exporting_cv.version])
+        exporting_cvv_version = entities.ContentViewVersion(id=exporting_cvv_id).read().version
+        ContentView.version_export(
+            {'export-dir': '{}'.format(self.export_base), 'id': exporting_cvv_id})
+        exported_tar = '{0}/export-{1}-{2}.tar'.format(
+            self.export_base, exporting_cv.name, exporting_cvv_version)
+        result = ssh.command("[ -f {0} ]".format(exported_tar))
+        self.assertEqual(result.return_code, 0)
+        exported_packages = Package.list({'content-view-version-id': exporting_cvv_id})
+        self.assertTrue(len(exported_packages) > 0)
+        self.set_importing_org(
+            prescene_dict['exporting_prodname'], prescene_dict['exporting_reponame'],
+            exporting_cv.name)
+        ContentView.version_import({
+            'export-tar': exported_tar,
+            'organization-id': self.importing_org.id
+        })
+        importing_cvv = self.importing_cv.read().version
+        self.assertTrue(len(importing_cvv) >= 1)
```

Comment:
Just have query here , In which phase C.V.V. would be `>1` ?

Reply:
Never :) , keeping it to 1.